src/components/authentication/LoginForm.vue

```diff
@@ -0,0 +1,125 @@
+<script setup>
+import { useForm } from 'vee-validate'
+import * as yup from 'yup'
+import { ref, computed } from 'vue'
+import { useUserStore } from '../../stores/userStore'
+
+const schema = yup.object({
+  email: yup.string().email().required(),
+  password: yup.string().min(6).required()
+})
+
+const { defineField, errors, handleSubmit } = useForm({
+  validationSchema: schema
+})
+
+const [email, emailAttrs] = defineField('email', {
+  validateOnModelUpdate: false
+})
+const [password, passwordAttrs] = defineField('password', {
+  validateOnModelUpdate: false
+})
+
+const onSubmit = handleSubmit((values) => {
+  const userStore = useUserStore()
+  const { email, password } = values
+  const result = userStore.login(email, password)
+  result ? emit('success') : emit('failure')
+})
+
+const emit = defineEmits(['success', 'failure', 'changeModal', 'inputStart'])
+</script>
+
+<template>
+  <div class="autho-modal__content login-container">
+    <h1 class="text-xl login-container__title">Welcome</h1>
+    <p class="text-sm login-container__description">Sign in to continue</p>
+    <form @submit="onSubmit" class="login-container__form login-form">
+      <input
+        class="login-form__input"
+        v-model="email"
+        v-bind="emailAttrs"
+        name="email"
+        type="email"
+        placeholder="Email"
+        @focus="$emit('inputStart')"
+      />
+      <span class="login-form__error">{{ errors.email }}</span>
+
+      <input
+        class="login-form__input"
+        v-model="password"
+        v-bind="passwordAttrs"
+        name="password"
+        placeholder="Password"
+        type="password"
+        @focus="$emit('inputStart')"
+      />
+      <span class="login-form__error">{{ errors.password }}</span>
+
+      <button class="login-form__submit-button">Log In</button>
+    </form>
```

Comment:
a shorter login-container__toggle-button for this button element maybe?

Reply:
This one is supposed to be submitting the information to the store and then to backend, its not the toggle